test/multipliers/VotingStreakMultiplier.t.sol

```diff
@@ -0,0 +1,338 @@
+import "forge-std/StdJson.sol";
+import {Test} from "forge-std/Test.sol";
+import {ERC20VotesUpgradeable} from
+    "openzeppelin-contracts-upgradeable/contracts/token/ERC20/extensions/ERC20VotesUpgradeable.sol";
+import {Ownable2StepUpgradeable} from "openzeppelin-contracts-upgradeable/contracts/access/Ownable2StepUpgradeable.sol";
+import {TransparentUpgradeableProxy} from
+    "openzeppelin-contracts/contracts/proxy/transparent/TransparentUpgradeableProxy.sol";
+
+import "script/Constants.s.sol";
+import {ButteredBread} from "src/ButteredBread.sol";
+import {IMultiplier} from "src/interfaces/IVotingMultipliers.sol";
+import {VotingMultipliers, IVotingMultipliers} from "src/VotingMultipliers.sol";
+import {VotingStreakMultiplier} from "src/multipliers/VotingStreakMultiplier.sol";
+import {YieldDistributor, IYieldDistributor} from "src/YieldDistributor.sol";
+import {YieldDistributorTestWrapper} from "src/test/YieldDistributorTestWrapper.sol";
+
+abstract contract Bread is ERC20VotesUpgradeable, Ownable2StepUpgradeable {
+    function claimYield(uint256 amount, address receiver) public virtual;
+    function yieldAccrued() external view virtual returns (uint256);
+    function setYieldClaimer(address _yieldClaimer) external virtual;
+    function mint(address receiver) external payable virtual;
+}
+
+contract VotingStreakMultiplierTest is Test {
```

Comment:
There is already a Voting multiplier testing section [here](https://github.com/BreadchainCoop/breadchain/blob/952757e85598922624ddf0d9f4bb363c04e72f25/test/YieldDistributor.t.sol#L454) , would probably be better to refactor this to extend that area

not gonna block this PR on that though tbh, thoughts @bagelface ? 

Lets keep in mind this will all be refactored for breadkit anyways